### PR TITLE
Clean icons

### PR DESCRIPTION
**Pull request by giardiv**
- [x] remove the icons from /sources folder
- [x] remore fontawesome from quartz
- [x] remove components/icons
- [x] remove components/typography/input-validation/icons
- [x] support icon color and sizes

### CODE REVIEW

**Comment by giardiv** [comment]
Task linked: [CU-1x73an0 Delete the sources folder and its icons in Quartz](https://app.clickup.com/t/1x73an0)

File: src/components/icon/index.tsx [hunk 1]
```diff
@@ -1,22 +1,30 @@
...
 export interface IconProps extends Omit<BoxProps, 'css'> {
-  size?: SizeProp;
-  icon: IconProp;
-  variant?: 'primary';
+  icon: IconName;
+  size?: IconSize;
+  color?: Colors;
+}
+
+export enum IconColors {
```
Comment (snqb) [inline src/components/icon/index.tsx]: what do we need this for?

File: src/components/file-loader/index.tsx [hunk 2]
```diff
@@ -60,7 +61,7 @@ const FileLoader: FC<QuartzFileLoaderProps> = ({
           removeHandler(id);
         }}
       >
-        <Flex>{icons.cross}</Flex>
+        <Flex><Icon size="sm" icon={IconName.cross} /></Flex>
```
Comment (snqb) [inline src/components/file-loader/index.tsx:64]: nit: this should be self-enclosing

**Review by snqb — CHANGES_REQUESTED** [review]
good besides `package-lock.json`

**Review by snqb — APPROVED** [review]

**Comment by giardiv** [comment]
Waiting for dependencies updates to be merged @snqb will check it out by the end of the week

**Comment by snqb** [comment]
Done in https://github.com/logicalclocks/quartz/pull/240

**Pull request closed without merging**